docs/schema.md

```diff
@@ -0,0 +1,121 @@
+
+> For the rest of this guide, we will assume that every action occur from the `opencerts-document` folder
+
+# Wrapping OpenCerts document
+
+`open-attestation-cli` provides a **--schema** option that we will make use of to create valid OpenCerts document.
+
+## Creating unwrapped document
+
+Create a document called `opencerts.json` in the `unwrapped` folder with the following content:
+
+```json
+{
+  "id": "53b75bbe",
+  "name": "Govtech Demo Certificate",
+  "description": "Govtech Demo Certificate",
+  "issuedOn": "2019-05-29T00:00:00+08:00",
+  "admissionDate": "2017-08-01T00:00:00+08:00",
+  "graduationDate": "2022-08-01T00:00:00+08:00",
+  "$template": {
+    "name": "GOVTECH_DEMO",
+    "type": "EMBEDDED_RENDERER",
+    "url": "https://demo-renderer.opencerts.io"
+  },
+  "issuers": [
+    {
+      "name": "Govtech",
+      "url": "https://tech.gov.sg",
+      "documentStore": "0x007d40224f6562461633ccfbaffd359ebb2fc9ba",
+      "identityProof": {
+        "type": "DNS-TXT",
+        "location": "demo.openattestation.com"
+      }
+    }
+  ],
+  "recipient": {
+    "name": "Your Name",
+    "nric": "SXXXXXXXY",
+    "course": "Govtech Demo"
+  },
+  "transcript": [
+    {
+      "name": "Introduction to Programming",
+      "grade": "A+",
+      "courseCredit": "3",
+      "courseCode": "CS 1110",
+      "examinationDate": "2017-12-01T00:00:00+08:00",
+      "semester": "1"
+    },
+    {
+      "name": "Object Oriented Programming in Java",
+      "grade": "A+",
+      "courseCredit": "4",
+      "courseCode": "CS 2110",
+      "examinationDate": "2017-12-01T00:00:00+08:00",
+      "semester": "1"
+    },
+    {
+      "name": "Microeconomics",
+      "grade": "A+",
+      "courseCredit": "4",
+      "courseCode": "ECON 3030",
+      "examinationDate": "2018-05-01T00:00:00+08:00",
+      "semester": "2"
+    },
+    {
+      "name": "Macroeconomics",
+      "grade": "A",
+      "courseCredit": "4",
+      "courseCode": "ECON 3040",
+      "examinationDate": "2018-05-01T00:00:00+08:00",
+      "semester": "2"
+    },
+    {
+      "name": "Econometrics",
+      "grade": "A-",
+      "courseCredit": "4",
+      "courseCode": "ECON 3120",
+      "examinationDate": "2018-05-01T00:00:00+08:00",
+      "semester": "2"
+    }
+  ],
+  "additionalData": {
+    "merit": "Y",
+    "studentId": "123456",
+    "transcriptId": "001"
+  }
+}
+```
+
+> Don't forget to replace `$template.url`, `issuers[0].documentStore` and `issuers[0].documentStore.identityProof.location` as explained in OpenAttestation documentation
+
+Run the cli using the schema option:
+
+```json
+❯  ./open-attestation-linux wrap unwrapped wrapped --schema https://schema.opencerts.io/transcripts/2.0
+✔  success   Batch Document Root: 0xc5507674eb34c36343d0da6a79a76c7967c5f3b1f7642c74ea822e7cff1b8a69
+```
+
```

Comment:
maybe something like `open-attestation-linux wrap` accepts 2 arguments <input folder> <output folder>, so a typical operation looks like this `./open-attestation-linux wrap input output`

Reply:
Added a note